test/file-concurrently.test.js

```diff
@@ -454,6 +527,116 @@ describe('download concurrently', function () {
             console.log(e);
         }
     }).timeout(10000);
+    it('status-503-one-chunk-fails', async function () {
+        nock('http://test-status-503', {
+            reqheaders: {
+                'range': 'bytes=0-0'
+            },
+        })
+            .get('/path/to/file.ext')
+            .reply(200, 'c', {
+                'content-type': 'text/plain',
+                'content-length': 1
+            });
+        // `e` fails at first with 503
```

Comment:
do you mean 'a' ?

Reply:
yep, had used `hello` then switched to a 3 letter word `cat` instead